pallets/investments/src/tests.rs

```diff
@@ -1289,11 +1289,11 @@ fn fulfillment_partially_works_low_price() {
 		{
 			assert_eq!(
 				free_balance_of(investment_account(INVESTMENT_0_0), AUSD_CURRENCY_ID),
-				207245508982035928145
+				207245508982035928140
```

Comment:
Rounding issues due to modifying `Rate` (27 decimals) to `Quantity` (18 decimals).